crates/uv/Cargo.toml

```diff
@@ -119,7 +119,7 @@ ignored = [
 ]
 
 [features]
-default = ["python", "pypi", "git", "performance"]
+default = ["python", "pypi", "git", "performance", "external"]
```

Comment:
I think this should be called `crates.io` or something, to match `pypi`. PyPI is also technically "external".

Reply:
Agree, I was surprised by this when reading.